src/components/api/codenvy-user.factory.js

```diff
@@ -58,9 +58,6 @@ export class CodenvyUser {
     // user roles
     this.isUserInRoleMap = new Map();
 
-    // fetch the user when we're initialized
```

Comment:
do we need all the refetchUser /fetchUser() method as part of codenvyUser as we've them in cheUser ?